addons/web/static/src/core/position_hook.js

```diff
@@ -274,14 +274,13 @@ export function reposition(popper, target, options, iframe) {
     // Get best positioning solution and apply it
     const position = getBestPosition(popper, target, options, iframe);
     const { top, left, direction, variant } = position;
-    popper.style.top = `${top}px`;
-    popper.style.left = `${left}px`;
+    popper.style.top = top > 0 ? `${top}px` : `${0}px`;
+    popper.style.left = left > 0 ? `${left}px` : `${0}px`;
```

Comment:
Is this working with the right position?
why {0} ?

Reply:
> Is this working with the right position?
I'm not really sure in what situations we want to have a negative position value for our fixed element- this will fix the popper to the top right right corner of the screen if it otherwise would be getting cutoff.